magento/magento2
# Add fulltext/keyword search field for product admin grid

### Description
This adds a keyword search field (like on sales_order_grid) to catalog_product_grid (product_listing)

### Manual testing scenarios
1. Log into the backend
2. Go to Catalog -> Products
3. Notice the key word search field
4. Enter arbitrary search terms occurring in product attributes
5. Apply and filter grid to only show matching products

### Fixes issues
1. magento/magento2#5785

### Contribution checklist
